src/project/FileTreeView.js

```diff
@@ -478,7 +478,42 @@ define(function (require, exports, module) {
         render: function () {
             var fullname = this.props.name,
                 extension = LanguageManager.getCompoundFileExtension(fullname),
-                name = _getName(fullname, extension);
+                name = _getName(fullname, extension),
+                fileType = "default";
+
+            switch (extension.toLowerCase()) {
+              case 'htmls':
+              case 'htm':
+              case 'htx':
+              case 'md':
+              case 'markdown':
+              case 'html':
```

Comment:
Please add cases for `'xml'`, and `'xhtml'` here too, and they can also use `"html"` as a type.